engines/asylum/puzzles/boardyouth.cpp

```diff
@@ -127,7 +127,32 @@ static const PuzzleBoard::PuzzleData puzzleYouthData[] = {
 		true,
 		10, 0,
 		"\xF8 \xF2 \xE5 \xF0 \xE4   \xFA \xE0 "
-	}
+	},
+	// Basque
+	{
+		55,
+		kGameFlag282,
+		431,
+		2,
+		{{3, false}, {4, false}, {0, false}},
+		9,
+		{
+			{'A',   32,  56},
+			{'T',   81,  68},
+			{'K',  136, 102},
```

Comment:
The x-coordinate is wrong here. Should be 124

Reply:
in the K?

Reply:
Yes, currently it points to 'e' in 'Kendu'